src/components/dialogs/substation-creation-dialog.js

```diff
@@ -0,0 +1,273 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import { InputBase, TextField } from '@material-ui/core';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Grid from '@material-ui/core/Grid';
+import { makeStyles, withStyles } from '@material-ui/core/styles';
+import { useSnackbar } from 'notistack';
+import PropTypes from 'prop-types';
+import React, { useCallback, useState } from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { useParams } from 'react-router-dom';
+import {
+    displayErrorMessageWithSnackbar,
+    useIntlRef,
+} from '../../utils/messages';
+import { createSubstation } from '../../utils/rest-api';
+import { validateField } from '../util/validation-functions';
+import { Autocomplete } from '@material-ui/lab';
+import { useParameterState } from '../parameters';
+import { PARAM_LANGUAGE } from '../../utils/config-params';
+import { getComputedLanguage } from '../../utils/language';
+
+const useStyles = makeStyles((theme) => ({
+    helperText: {
+        margin: 0,
+        marginTop: 4,
+    },
+    popper: {
+        style: {
+            width: 'fit-content',
+        },
+    },
+    h3: {
+        marginBottom: 0,
+        paddingBottom: 1,
+    },
+    h4: {
+        marginBottom: 0,
+    },
+}));
+
+const BootstrapInput = withStyles(() => ({
+    input: {
+        minWidth: '223px',
+        minHeight: '19px',
+        color: 'white',
+        borderColor: 'grey',
+        borderRadius: 4,
+        position: 'relative',
+        border: '1px solid',
+        fontSize: 16,
+        paddingLeft: 14,
+        paddingRight: 14,
+        paddingTop: 18.5,
+        paddingBottom: 18.5,
+    },
```

Comment:
How did you choose these values ? I tried to modify these values, but it seems to have no effect on the way the component using this style is displayed. I think BootstrapInput can be removed.